system_tests/tls_system_test.go

```diff
@@ -0,0 +1,187 @@
+package system_tests
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/google/uuid"
+	rabbitmqv1beta1 "github.com/rabbitmq/cluster-operator/api/v1beta1"
+	topology "github.com/rabbitmq/messaging-topology-operator/api/v1alpha2"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Management over TLS", func() {
```

Comment:
I would say "RabbitMQ with TLS"? Mentioning management endpoint might be a bit detailed here :)